pages/signin.vue

```diff
@@ -0,0 +1,245 @@
+                    </b-form-invalid-feedback>
+                </b-form-group>
+
+                <b-form-group id="passwordGroup">
+                    <b-form-input
+                            placeholder="password"
+                            id="signInPassword"
+                            name="signInPassword"
+                            type="password"
+                            v-model="$v.form.password.$model"
+                            :state="$v.form.password.$dirty ? !$v.form.password.$error : null"
+                            aria-describedby="signInPassword-live-feedback"
+                    ></b-form-input>
+
+                    <b-form-invalid-feedback id="signInPassword-live-feedback">
+                        This is a required field.
+                    </b-form-invalid-feedback>
+                </b-form-group>
+
+                <b-form-group id="passwordConfirmGroup">
+                    <b-form-input
+                            placeholder="passwordConfirm"
+                            id="signInPasswordConfirm"
+                            name="signInPasswordConfirm"
+                            type="password"
+                            v-model="$v.form.passwordConfirm.$model"
+                            :state="$v.form.passwordConfirm.$dirty ? !$v.form.passwordConfirm.$error : null"
+                            aria-describedby="signInPasswordConfirm-live-feedback"
+                    ></b-form-input>
+
+                    <b-form-invalid-feedback id="signInPasswordConfirm-live-feedback">
+                        Passwords must be identical.
+                    </b-form-invalid-feedback>
+                </b-form-group>
+
+                <b-button type="submit" variant="primary" :disabled="$v.form.$invalid">Submit</b-button>
+            </b-form>
+            <nuxt-link to="login">
+                J'ai déjà un compte
+            </nuxt-link>
+        </div>
+    </div>
+</template>
+
+<script>
+  import { validationMixin } from 'vuelidate';
+  import { required, email, sameAs, maxLength } from 'vuelidate/lib/validators';
+
+  export default {
+    mixins: [validationMixin],
+    auth: false,
+    data() {
+      return {
+        user: {},
+        form: {
+          firstName: null,
+          lastName: null,
+          email: null,
+          password: null,
+          passwordConfirm: null
+        }
+      }
+    },
+    validations: {
+      form: {
+        firstName: {
+          required
+        },
+        lastName: {
+          required
+        },
+        email: {
+          required,
+          email,
+          maxLength: maxLength(99)
+        },
+        password: {
+          required
+        },
+        passwordConfirm: {
+          required,
+          sameAsPassword: sameAs('password')
+        }
+      }
+    },
+    methods: {
+      async signIn() {
+        const params = {
```

Comment:
We should maybe give a better name to params so it will be more clear when we read it ? userParams, formParams, etc. ?